### ISSUE TITLE
Beta - Permissions on family page.

### ISSUE THREAD

**Issue by makitso** [fisharebest/webtrees#4157]
I am a bit curious.  The link below takes you to one of my relative, the family tab you can see his marriage to Margaret Ihde.    They had 4 children one of which is still alive.  As such the person block is marked as Private.  

https://skatekey.net/tree/peters/individual/I0006/Cornelis-Hendrikus-Peters#tab-relatives

Now, select the  Facts and events tab and select the link under, Family navigator, "Family with Margaret Ihde".  The link is 

https://skatekey.net/tree/peters/family/F010/Cornelis-Hendrikus-Peters-Margaret-Ihdet

Which gives a 403 error and.   "This family does not exist or you do not have permission to view it."

Shouldn't  this page have been displayed with the child person block marked as private?

**Comment by fisharebest** [comment]
> As such the person block is marked as Private.

I guess you have selected the opion "show private relationships"?

You can see the relationship, but not the record.

**Comment by makitso** [comment]
Yes, I do have the Show Private Relationships set to "Show Visitors."

My point here is that if a visitor can see the relationships for Cornelis Hendrikus Peters on the Decendants Chart and the Family navigator,  why can't the visitor see the relationships on Family with xxxx ?  Seems like an inconsistent application of the  "show privacy relationships" setting.
